src/application/window.rs

```diff
@@ -787,18 +802,24 @@ pub(crate) fn hit_test(window: &gdk::Window, cx: f64, cy: f64) -> WindowEdge {
   };
 
   // FIXME: calling `window.begin_resize_drag` seems to revert the cursor back to normal style
-  window.set_cursor(Some(&Cursor::new_for_display(
-    &display,
-    match edge {
-      WindowEdge::North | WindowEdge::South => CursorType::SbVDoubleArrow,
-      WindowEdge::East | WindowEdge::West => CursorType::SbHDoubleArrow,
-      WindowEdge::NorthWest => CursorType::TopLeftCorner,
-      WindowEdge::NorthEast => CursorType::TopRightCorner,
-      WindowEdge::SouthEast => CursorType::BottomRightCorner,
-      WindowEdge::SouthWest => CursorType::BottomLeftCorner,
-      _ => CursorType::LeftPtr,
-    },
-  )));
-
+  if edge != WindowEdge::__Unknown(8) {
+    window.set_cursor(
+      Cursor::from_name(
```

Comment:
What's the reason to use `from_name` instead of `new_for_display`.

Reply:
mainly for consistency in the codebase, and because it is much similar to winit's CursorIcon enum.

Also that `if` statement was a left over from unsuccessful experiment, I will remove it